getting-started/mirroring.md

```diff
@@ -0,0 +1,32 @@
+# Mirroring SystemLink Repositories
+
+SystemLink Enterprise is distributed in the form of Helm charts and Docker images. These resources are located in the following artifact repositories.
+
+Helm:   `https://niedge01.jfrog.io/artifactory/ni-helm`
+Docker: `https://niedge01.jfrog.io/artifactory/ni-docker`
+
+To create a local mirror for the SystemLink application, both must be replicated to a local artifact repository.
+
+## Repository Setup
+
+The NI artifact repositories are organized using the Virtual Repository layout described in JFrog's [whitepaper on structuring and naming repositories](https://jfrog.com/whitepaper/best-practices-structuring-naming-artifactory-repositories/). Any mirror must maintain the same relative naming and layout of artifacts. Failure to do so may prevent installation of the application.
+
+TODO: pull through proxy cache
```

Comment:
The critical piece of information here is that they configure their mirror such that anything referencing niedge01.jfrog.io is whitelisted as a source for the pull-through proxy.  Configuring this allows the mirror to recognize the "registry" (ie: `niedge01.jfrog.io/ni-docker`) as a remote location.  Since it doesn't exist on their mirror, it will pull-through their mirror as a `proxy` and cache the container on the mirror.  (hence:  pull-through proxy cache).  This allows pull-through behavior only for whitelisted sources, and still prevents pulling public images or from other repos.

With this enabled, the global registry override in the NI helm charts will work without needing to point specifically at the customer's artifactory.  This is what actually makes it a mirror instead of a manually maintained private repo.

This can also be configured for other 3rd-party repositories such as @bitnami, @strimzi, @elyra, or @mongoclient, etc.  This way, 3rd-party charts that reference these other image repositories don't need to explicitly implement the global override that our NI (and @bitnami) charts do.  They can be linked together as virtual repositories, and everything can reside under the .../ni-docker path.

Reply:
I took a stab at describing this. I did not mention mirroring additional repos. Our current plan is to use the global registry for all dependencies.